### ISSUE TITLE
Remove from_array and to_array from Cubic class

### ISSUE THREAD

**Issue by jgostick** [PMEAL/OpenPNM#1693]
These methods are actually broken at present since the `'pore.index'` array is not longer found on the object.  I think it's safe to remove them.   

It's also quite easy to do the `from_array` bit, just do `pn['pore.blah'] = array.flatten()`.  

The reverse is a bit more involved:  `arr = pn['pore.blah'].reshape(pn._shape)`.

The above gets tricky when the boundary pores or other topology mangling has been done, but in that case to/from_array would be less useful

There is also the case of the CubicTemplate, but this is also easy:

``pn['pore.blah'] = array[pn['pore.template_indices']`.  

Basically, I think these methods can go.

**Comment by jgostick** [comment]
This was done on PR #2054.  No changes to the examples or tests were required because it was not used.
